src/destreamer.ts

```diff
@@ -347,6 +347,31 @@ async function downloadVideo(videoGUIDs: Array<string>,
         logger.info(`Video no.${videos.indexOf(video) + 1} downloaded!!\n\n`);
     }
 
+    // START CONCATENATE
+    if(argv.concat){
+        logger.info("Concatenating videos...");
+        let concatFiles = "";
+        let concatName = `videos_concatenated ${videos.length}`;
+        for(const video of videos){
+            concatFiles += ("file '" + video.outPath + "'\n");
+            logger.info("file " + video.outPath);
+        }
+
+        // Create a file containg all videos path
+        await fs.promises.writeFile('files.txt', concatFiles);
```

Comment:
use [fs.writeFileSync](https://nodejs.org/api/fs.html#fs_fs_writefilesync_file_data_options) for consistency with all the other code please

Reply:
For the moment I did this and fixed CommandLineParser. I will try to figure out how to work with stream and pipelines to merge file using "Nodejs vanilla" without ffmpeg